pkgs/top-level/haskell-packages.nix

```diff
@@ -95,6 +95,8 @@ in {
         # Musl bindists do not exist for ghc 8.6.5, so we use 8.10.* for them
         if stdenv.isAarch64 || stdenv.isAarch32 then
           packages.ghc8107BinaryMinimal
+        else if stdenv.hostPlatform.isPower64 && stdenv.hostPlatform.isLittleEndian then
+          packages.ghc865Binary
```

Comment:
Could you add a comment for these that there's no upstream bindist available for a bootstrap- suitable GHC version?

Reply:
Done.